thoth/glyph/cli.py

```diff
@@ -93,18 +102,79 @@ def generate(output: str) -> None:
     "--message",
     "-m",
     type=str,
-    default="-",
+    required=True,
     help="Commit message to be classified",
 )
 def classify(message: str) -> None:
     """Generate CHANGELOG entries from the current Git project."""
     _LOGGER.info("Hello, glyph!")
-    if message == "-":
-        click.echo("Please enter message")
+    _LOGGER.info("Model Path : " + DEFAULT_MODEL_PATH)
+    classifier = load_model(DEFAULT_MODEL_PATH) 
+    label = classifier.predict(message.lower())
+    click.echo("Label : " + str(label[0][0])[8:])
+
+@cli.command()
+@click.option(
+    "--path",
+    "-p",
+    type=str,
+    required=True,
+    help="Path to Git repository"
+)
+@click.option(
+    "--start",
+    type=str,
+    help="Starting date"
+)
+@click.option(
+    "--end",
+    type=str,
+    help="End date"
+)
+@click.option(
+    "--output",
+    type=str,
+    help="Generated output file"
+)
+@click.option(
+    "--model",
+    type=str,
+    help="Generated output file"
+)
+def classifyrepo(path: str, start: str, end: str, output: str, model:str) -> None:
+    if model is None:
+        model = DEFAULT_MODEL_PATH
+    _LOGGER.info("Model Path : " + model)
+    classifier = load_model(model)
+    start_time = 0
+    end_time = 4119206400
+
+    if start is not None:
+        start_time = int(time.mktime(datetime.datetime.strptime(start, "%Y-%m-%d").timetuple()))
+
+    if end is not None:
+        end_time = int(time.mktime(datetime.datetime.strptime(end, "%Y-%m-%d").timetuple()))
+
+    repo = Repository(path + "/.git")
+    orig_messages = []
+    for commit in repo.walk(repo.head.target, GIT_SORT_TOPOLOGICAL):
+        if(commit.commit_time > start_time and commit.commit_time < end_time):
+             orig_messages.append(commit.message.lower())
+
+    print(str(len(orig_messages)) + " commits classified")
+    df = pd.DataFrame(orig_messages, columns = ['message'])
+    df = df.replace('\n','', regex=True)
+    commits = list(df['message'].astype(str))
+    labels = classifier.predict(commits)
+    res = list(zip(*labels))
+    res_list = [x[0] for x in res]
+    lst2 = [item[0] for item in res_list]
+    df['labels_predicted'] = lst2
+    if output is None:
+        df.to_csv("output.csv", sep='\t')
```

Comment:
What about printing to stdout? Unix utils are designed in that way.

Reply:
I didn't quite get it.
Do you want the output printed in the console? or some other way to write the file?

Reply:
To the console, so that it can be used with pipes and such.

Reply:
Sure, I'll use the print() statement for printing to console when the path to output file is not provided

Reply:
Fixed and pushed